color/Cargo.toml

```diff
@@ -22,9 +22,15 @@ targets = []
 default = ["std"]
 std = []
 libm = ["dep:libm"]
+bytemuck = ["dep:bytemuck"]
 
 [dependencies]
 
+[dependencies.bytemuck]
+version = "1.19.0"
+optional = true
+default-features = false
```

Comment:
I'm sure there's a fallacy which applies to this argument, but nevertheless I'm going to make it. If our users want to use bytemuck, how likely is it that they also won't have the derive feature enabled?

That is, do we gain much value from being so austere?